addons/addon-environment-sc-api/packages/environment-type-mgmt-services/lib/environment-type/env-type-candidate-service.js

```diff
@@ -205,6 +205,10 @@ class EnvTypeCandidateService extends Service {
           _.filter(paResult.ProvisioningArtifactDetails || [], pa => pa.Active), // filter out inactive versions
           v => -1 * v.CreatedTime,
         );
+        if (_.isEmpty(provisioningArtifacts)) {
+          // No active versions
+          return null;
+        }
```

Comment:
Can we add a unit test test as well?

Reply:
yeah, normally that'd be ideal, but this class has zero tests and adding the test setup is a bit too much for an oncall task. I need to switch to other tickets/bugs.